src/tools/crossgen2/Common/TypeSystem/Common/Utilities/DebugNameFormatter.cs

```diff
@@ -216,6 +245,36 @@ protected override Void AppendNameForInstantiatedType(StringBuilder sb, DefType
             return Void.Value;
         }
 
+        protected override DefType GetContainingType(DefType possibleInnerType, FormatOptions options)
+        {
+            try
+            {
+                return possibleInnerType.ContainingType;
+            }
+            catch
+            {
+                return null;
+            }
+        }
+
+        private void GetDiagnosticName(GenericParameterDesc type, ref string diagnosticName)
```

Comment:
These two helpers don't seem to be needed anymore.

Reply:
Good point. Deleted